inst/components/scss/toolbar.scss

```diff
@@ -53,6 +59,29 @@
     }
   }
 
+  /* ---- Toolbar Divider ---- */
+
+  .bslib-toolbar-divider{
+    align-self: center; // Ensures divider lines stretch to full height
+    opacity: 0.75;
+    height: var(--_divider-height);
+    // Sets divider total width based on bslib gap variable if set, otherwise
+    // uses the toolbar gap default (--bslib-toolbar-gap isn't defined when
+    // --_toolbar-gap is originally set so we can't use the same strategy as
+    // height)
+    width: var(--bslib-toolbar-gap, var(--_toolbar-gap));
+
+      // Use a pseudo-element for the actual line so custom width controls spacing
+      &::before {
+        content: "";
+        display: block;
+        width: var(--bslib-toolbar-divider-width, var(--_divider-width));
+        height: 100%;
+        background-color: var(--bs-border-color);
```

Comment:
Maybe we'd want `--bs-border-color-translucent` instead of border color + opacity.